quetz/utils.py

```diff
@@ -21,3 +22,19 @@ def add_entry_for_index(files, subdir, fname, data_bytes):
             "sha256": sha.hexdigest(),
         }
     )
+
+
+class TicToc:
+    def __init__(self, description):
+        self.description = description
+
+    def __enter__(self):
+        self.start = time.time()
+
+    def __exit__(self, ty, val, tb):
+        self.stop = time.time()
+        print(f"[TOC] {self.description}: {self.stop - self.start}")
```

Comment:
logger.debug ?

Reply:
yeah I tried to get a logger here, but then all logs were printed twice. Do i need to get a logger with a different name?

Reply:
no, I think that's a bug of uvicorn, it was fixed upstream but we need to wait for a release